zilliqa/src/consensus.rs

```diff
@@ -1893,10 +1946,11 @@ impl Consensus {
                     new_high_qc_block_view + 1,
                     current_high_qc_view,
                 );
+                //TODO write high_qc and current_view in one db call
```

Comment:
Is it the case that we always call `set_view` before sending a `Proposal`/`View`/`NewView` for a given view?